soccer/Processor.cpp

```diff
@@ -295,10 +305,26 @@ void Processor::run() {
             log->CopyFrom(packet->wrapper);
 
             curStatus.lastVisionTime = packet->receivedTime;
+
+            if (packet->wrapper.has_geometry()) {
+                // DEMO: Test out field sizes
+                const SSL_GeometryFieldSize fieldSize = packet->wrapper.geometry().field();
+                // FIXME - Account for network latency
+
+                if (fieldSize.field_length() != 0 &&
```

Comment:
It would be best if we do a more thorough comparison here.  One potential area I could see this going wrong is if the length of the field at comp matches our default setup, but the field construction people screwed up the width (they always screw up something).  As is, we wouldn't detect and would just stick with our incorrect default width.

A good solution would be to implement `operator==()` on `Field_Dimensions` and use it here.